### PR TITLE
[DAT-487] - Integrate the "Contact Information" section with DopplerUsersApi

### PR DESCRIPTION
**Pull request by jhoffman-ms**
Integrated the "Contact Information" section with DopplerUsers api to get the Information.

**Task:** [DAT-487](https://makingsense.atlassian.net/jira/software/projects/DAT/boards/62?selectedIssue=DAT-487)

### CODE REVIEW

**Comment by cbernat** [comment]
I've tested this in int and it is not looking good https://cdn.fromdoppler.com/doppler-webapp/int-build3480/#/checkout/standard/2?promo-code=ALLPLANS
image
Am I missing something?

**Comment by jhoffman-ms** [comment]
> I've tested this in int and it is not looking good https://cdn.fromdoppler.com/doppler-webapp/int-build3480/#/checkout/standard/2?promo-code=ALLPLANS
> image
> Am I missing something?

@cbernat , @msimionati is resolving it. Thanks.

**Comment by jcamposmk** [comment]
image

**Comment by jhoffman-ms** [comment]
All feedback are done. Let me know if is ok to merge.

File: src/components/ChangePlan/Checkout/ContactInformation/ContactInformation.js [hunk 1]
```diff
@@ -80,6 +80,19 @@ export const ContactInformation = InjectAppServices(
       }
     };
 
+    const complete = (contactInformation) => {
```
Comment (cbernat) [inline src/components/ChangePlan/Checkout/ContactInformation/ContactInformation.js:83]: With this issue resolved this is ready to merge @jhoffman-ms 🎉

Comment (jhoffman-ms) [reply src/components/ChangePlan/Checkout/ContactInformation/ContactInformation.js]: Done

**Pull request merged**